GCRCatalogs/catalog_configs/photoz_SKRF_v1_for_cosmoDC2_v1.1.4_small.yaml

```
@@ -0,0 +1,6 @@
+# test for olivia photo-z's
+
+subclass_name: dc2_photoz_parquet.PZSKRFCatalog
+base_dir: /global/cfs/cdirs/lsst/shared/xgal/cosmoDC2/addons/photoz_v1.1.4/10_year_error_estimates/OLIVIA_WORK
```

Need to use the root_dir symbol here: 
```suggestion
base_dir: ^/xgal/cosmoDC2/addons/photoz_v1.1.4/10_year_error_estimates/OLIVIA_WORK
```

---

Whoops, forgot to modify to the relative path, made the suggested change.